events.md

```diff
@@ -164,6 +165,7 @@ meta:
 *  ``website_uri``: Where to find the website joind.in page for this event
 *  ``website_humane_uri``: Where to find the website joind.in page for this event, using the quicklink if there is one (depends whether the event has a stub)
 *  ``hosts``: The organisers for this event.  An array where each element gives both the ``host_uri`` of the user and a display name to use.  *See also* [users]({{ site.baseurl}}/users.html)
+*  ``can_edit``: Whether the current user can edit this event or not.  This field is only returned when there is a valid logged in user.
```

Comment:
Thinking... the `attending` field is present irrespective of logged-in state. Should `can_edit` be the same?